plugin-api.d.ts

```diff
@@ -1755,8 +1737,6 @@ interface Variable {
   readonly id: string
   name: string
   description: string
-  hiddenFromPublishing: boolean
-  getPublishStatusAsync(): Promise<PublishStatus>
```

Comment:
jw double checking: is it expected that these are removed when running the `npm run update-typings` script?

Reply:
Good question. It looks like `getPublishStatusAsync` should've stuck around... But `hiddenFromPublishing` seems like it isn't in our docs at the moment.

Reply:
Oh, yes this is correct. It's still included in https://github.com/figma/plugin-typings/blob/533d2ad08a95f7b6f2261b019c6ae54b58ad2b55/plugin-api.d.ts#L1308C1-L1308C50, which is where it's defined in our docs.

Reply:
I see, and these two have been removed from `interface Variable` and `interface VariableCollection` - this makes sense! https://github.com/figma/plugin-typings/blob/533d2ad08a95f7b6f2261b019c6ae54b58ad2b55/plugin-api.d.ts#L1736-L1770

Reply:
@RobbieZhuang @rohitchouhan8 Wait, why are we removing these properties? We definitely want them: https://github.com/figma/figma/pull/167538

Reply:
I was running the `npm run update-typings` script and it seems like these types aren't currently defined in our `plugin-docs`. My understanding is that we want to try to keep those as inline as possible? When Robbie adds the docs, then those typings can then get re-added.

Reply:
Yeah, I guess we're running into race conditions with multiple docs PRs in flight.

Reply:
It sounds like we'll need to run `npm run update-typings` for https://github.com/figma/figma/pull/167538 as well as for https://github.com/figma/figma/pull/167522 again next week once our docs changes are in?

@james04321 do our api changes depend on the types being defined here in `plugin-api.d.ts`? I thought https://github.com/figma/figma/pull/167522 originally failed a test because some of these types were missing